fpdcc/ccfp-asset-dashboard
# Fix funding delete bug

## Overview

- Closes #212
- Fixes issue found after merging PR #211 

### Demo
![phase-funding-fix](https://user-images.githubusercontent.com/38969506/182645112-60321f34-203b-4c36-ba00-6509c4cc8075.gif)


### Notes
The delete button used the pk for the phase and not the funding stream, so it led to a 404 not found.

I realized that the fix from PR #211 won't affect current scores. This fix works so it's backwards compatible, but should we maybe write a script to update the database and remove all `None` values from all ProjectScore rows?

## Testing Instructions
- visit review app tk
- go to a phase
- delete funding
- make sure it redirects to a confirmation screen, and then back to the correct phase
